sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -131,6 +132,34 @@ public partial class ModelReaderWriterOptions
         public static System.ClientModel.Primitives.ModelReaderWriterOptions Json { get { throw null; } }
         public static System.ClientModel.Primitives.ModelReaderWriterOptions Xml { get { throw null; } }
     }
+    public partial class MultipartBinaryContent : System.ClientModel.BinaryContent, System.IDisposable
+    {
+        public MultipartBinaryContent() { }
+        public MultipartBinaryContent(string subtype) { }
+        public MultipartBinaryContent(string subtype, string boundary) { }
+        public string Boundary { get { throw null; } }
+        public string ContentType { get { throw null; } }
+        public void Add(System.ClientModel.BinaryContent part) { }
```

Comment:
We want to add strings and streams directly so we can use the BCL type under the hood